### PR TITLE
log2text.py: added --min-level

### PR DESCRIPTION
**Pull request by lior-stratoscale**

### CODE REVIEW

File: py/strato/common/log/log2text.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecon
         self._firstClock = None
         self._clock = self._relativeClock if relativeTime else self._absoluteClock
         self._relativeClockFormat = "%.6f" if microsecondPrecision else "%.3f"
-        self._minimumLevel = logging.INFO if noDebug else logging.DEBUG
+        self._minimumLevel = self._min_level(min_level, noDebug)
```
Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: why would you pass both of the parameters and not decide on min-level before the init? think that would be cleaner

File: py/strato/common/log/log2text.py [hunk 2]
```diff
@@ -427,7 +443,7 @@ def executeRemotely(args):
     formatter = Formatter(
         noDebug=args.noDebug, relativeTime=args.relativeTime, noColors=args.noColors,
         microsecondPrecision=args.microsecondPrecision, showFullPaths=args.showFullPaths,
-        withThreads=args.withThreads, utc=args.utc)
+        withThreads=args.withThreads, utc=args.utc, min_level=args.min_level)
```
Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: camel case it please

File: py/strato/common/log/log2text.py [hunk 3]
```diff
@@ -425,7 +442,7 @@ def executeRemotely(args):
         sys.exit(result)
 
     formatter = Formatter(
-        noDebug=args.noDebug, relativeTime=args.relativeTime, noColors=args.noColors,
+        minimumLevel=min_level(args.min_level, args.noDebug), relativeTime=args.relativeTime, noColors=args.noColors,
```
Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: camel case

**Comment by sergey-stratoscale** [comment]
ship it

File: py/strato/common/log/log2text.py [hunk 4]
```diff
@@ -396,11 +396,28 @@ def executeRemotely(args):
...
+            return level
+    return logging.DEBUG
+
+
 if __name__ == "__main__":
     import argparse
     parser = argparse.ArgumentParser()
     parser.add_argument("logFiles", metavar='logFile', nargs='*', help='logfiles to read')
     parser.add_argument('-d', "--noDebug", action='store_true', help='filter out debug messages')
+    parser.add_argument('-l', '--min-level', action='store', default='', help='minimal log level to display (substring is OK, case-insensitive)')
```
Comment (adir-stratoscale) [inline py/strato/common/log/log2text.py]: curios: why not defining `-v / --verbose` with `action=count` (see: https://docs.python.org/2/library/argparse.html#action)? 
this way every occurrence of `-v` will increase the logging level to display.

**Pull request merged**